symplyphysics/laws/thermodynamics/work_done_by_gas_in_isobaric_process.py

```diff
@@ -0,0 +1,34 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+# Work done by gas: A = p * (V_2 - V_1), where
```

Comment:
This law looks quite simple. Do you want to try to prove it?

Reply:
Take some more general law and transform it until it matches the law to prove. Eg https://github.com/blackyblack/symplyphysics/blob/ea325837f3684750a2fa1b4e68af4d3966a1960c/symplyphysics/laws/dynamics/period_of_spring_from_mass.py#L40